clients/privacy-center/packages/fides-consent/README.md

```diff
@@ -85,24 +111,56 @@ if (Fides.consent.data_sales) {
 }
 ```
 
+### Meta Pixel
+
+
+To integrate with [Meta's tracking pixel](https://developers.facebook.com/docs/meta-pixel/get-started),
+call `Fides.meta(options)`, where options is an with the following properties:
+
+- `consent`: boolean - Whether the user consents to [Meta's use of cookies](https://developers.facebook.com/docs/meta-pixel/implementation/gdpr). 
+  If `true`, consent it granted. Otherwise, consent is revoked.
+- `dataUse`: boolean - Whether user's data can be [used by Meta](https://developers.facebook.com/docs/meta-pixel/implementation/ccpa).
+  If `true`, Meta may use the data. Otherwise, Meta's Limited Data Use (LDU) mode will be be applied
+  as if the user were in California.
```

Comment:
Similarly, whack-a-mole with CCPA. But really if you want to prevent data-sales-ish behavior, it seems you need to tell FB the user is in CA.